src/NUnitTestAdapterTests/NUnit3TestAdapterTests.csproj

```diff
@@ -38,7 +38,7 @@
   <ItemGroup>
     <Reference Include="Microsoft.VisualStudio.TestPlatform.ObjectModel">
       <SpecificVersion>False</SpecificVersion>
-      <HintPath>$(MSBuildProgramFiles32)\Microsoft Visual Studio 14.0\Common7\IDE\CommonExtensions\Microsoft\TestWindow\Microsoft.VisualStudio.TestPlatform.ObjectModel.dll</HintPath>
+      <HintPath>$(MSBuildProgramFiles32)\Microsoft Visual Studio 12.0\Common7\IDE\CommonExtensions\Microsoft\TestWindow\Microsoft.VisualStudio.TestPlatform.ObjectModel.dll</HintPath>
```

Comment:
You changed this to 11.0 in the other project. I guess you have 2012 and 2013 installed? Once again minor for now, but I would like to see what we can do with this. In my GitHub extension, I used a NuGet package to pull the VS assemblies in making it much more version tolerant. We should see if we can do the same here.